Octokit.Reactive/Clients/IObservableOrganizationsClient.cs

```diff
@@ -15,6 +15,11 @@ public interface IObservableOrganizationsClient
         /// </summary>
         IObservableTeamsClient Team { get; }
 
+        /// <summary>
+        /// Returns a client to manage discussions for a team.
+        /// </summary>
+        IObservableTeamDiscussionsClient TeamDiscussions { get; }
```

Comment:
This sub client should be put under `TeamsClient` rather than `OrganizationsClient`, based on the API docs structure:

![image](https://user-images.githubusercontent.com/5425163/48131596-616cc180-e2dc-11e8-9f35-0710643ed7aa.png)

Also can we please make the property should be singular and just be called `Discussion`?  We would access it like  `client.Team.Discussion.Get()`

Reply:
Moved to `IObservableTeamsClient`